crypto/evp/evp_utils.c

```diff
@@ -18,7 +18,7 @@
 #include "evp_local.h"
 
 /*
- * EVP_CTRL_RET_UNSUPPORTED = -1 is the returned value from any ctrl function
+ * OSSL_RET_UNSUPPORTED = -2 is the returned value from any ctrl function
```

Comment:
Regardless of whether `EVP_CTRL_RET_UNSUPPORTED` had the wrong value or not, don't you break applications by returning a different value now?